### PR TITLE
fix: bump extension docs to v6

### PR DESCRIPTION
**Pull request by nicolethoen**

### CODE REVIEW

**Comment by andrew-ronaldson** [comment]
I noticed the log viewer extension has an option to toggle dark mode.
It didn't seem to work but the dark theme switcher in the masthead does.

**Comment by nicolethoen** [comment]
I will open a follow up issue with the log-viewer. The dark theme should be able be enabled even when the rest of the app is in light theme. It's probably an issue with how the tokens were updated.

**Comment by nicolethoen** [comment]
I'll open a bug for user-feedback. That was not introduced with this PR. It's already broken on staging.

**Comment by nicolethoen** [comment]
Bug fix issue for user feedback: https://github.com/patternfly/react-user-feedback/issues/85

**Review by dlabaj — APPROVED** [review]

File: packages/documentation-framework/versions.json [hunk 1]
```diff
@@ -6,23 +6,24 @@
...
+        "@patternfly/react-topology": "6.0.1",
+        "@patternfly/react-user-feedback": "6.0.0",
+        "@patternfly/react-console": "6.0.0",
+        "@patternfly/quickstarts": "6.0.0",
+        "@patternfly/react-virtualized-extension": "6.0.0",
+        "@patternfly/react-templates": "6.0.0",
+        "@patternfly/virtual-assistant": "2.0.0"
       }
     },{
       "name": "prerelease",
```
Comment (kmcfaul) [inline packages/documentation-framework/versions.json:29]: Couple lines down can we flip the prerelease both to `hidden`? Or remove it.

**Review by kmcfaul — APPROVED** [review]

**Pull request merged**